FF1Lib/Hacks.cs

```diff
@@ -2557,5 +2557,40 @@ public void NoItemMagic(Flags flags)
 
 			WriteText(itemnames, FF1Rom.ItemTextPointerOffset, FF1Rom.ItemTextPointerBase, FF1Rom.ItemTextOffset, FF1Rom.UnusedGoldItems);
 		}
+
+		public void HackMinimap(OverworldMap map) {
+		    var compresedMap = map.GetCompressedMapRows();
+		    var decompressedMap = map.DecompressMapRows(compresedMap);
+		    byte emptyTile = 0xff;

```

Comment:
I'm getting super nitpicky but this _could_ be a small helper function called findEmptyTile that returns as soon as it finds one, so that you don't have to have all the funky checks at each loop iteration.

`const emptyTile = findEmptyTile(decompressedMap);`

Reply:
I cleaned it up.